.github/workflows/nightly.yml

```diff
@@ -0,0 +1,32 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '45 8 * * *'
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Tag name to trigger the release'
+        required: false
+        default: 'nightly'
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.LYCHEE }}
```

Comment:
Can we rename that to `NIGHTLY_BUILD_TOKEN` or something to make the use-case more explicit?